packages/manager/apps/web/client/app/hosting/translations/Messages_fr_FR.json

```
@@ -1398,6 +1398,7 @@
   "hosting_dashboard_access": "Accès",
   "common_api_hostingWeb-1": "Vous ne pouvez pas migrer vers cette offre car votre offre actuelle dispose de plus d'e-mails que l'offre choisie.",
   "hosting_dashboard_service_activate_email": "Activer mon offre e-mail",
+  "hosting_dashboard_service_detach_email_option": "Délier mon option e-mail",
```

```suggestion
  "hosting_dashboard_service_detach_email_option": "Délier option e-mail de mon hébergement",
```

To reduce doubt in the users mind ?

I mean, if I don't know much about how OVH work, I don't know what `unlinking an email address` does 🤔

---

@FredericEspiau more details of this action are written in the section page (once user has clicked and is redirected)